step-templates/releasenoteshub-pull-trigger.json

```diff
@@ -0,0 +1,83 @@
+{
+  "Id": "8acee70f-0b6d-43a8-ab16-b832b37e0422",
+  "Name": "ReleaseNotesHub  Trigger",
+  "Description": "ReleaseNotesHub is a B2B\\B2C SaaS solution that automates the generation and publication of release notes. With RnHub the creation of Release Notes becomes another step within your CI/CD pipeline. RnHub can pull content from many systems including TFS, Azure Devops, GitHub, Jira and Asana. \n\nUse this step to trigger a pull from your Octopus Deploy process. Visit [here](https://support.releasenoteshub.com/article/show/111570-how-to-trigger-a-pull-from-octopus-deploy) for setup guide. \n\nhttps://www.releasenoteshub.com\n",
+  "ActionType": "Octopus.Script",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptBody": "[system.version]$v \n\nif ( -not [string]::IsNullOrEmpty( $Version ) )\n{\n\t$v = New-Object System.Version($Version)\n}\nelse\n{\n\t$v = New-Object System.Version($OctopusParameters['Octopus.Release.Number'])\n}\n\n$headers = @{\n    \"Authorization\" = \"ApiKey $ApiKey\"\n}\n\n$body = @{\n    \"version\" = @{\"VersionMajor\" = $v.Major; \"VersionMinor\" = $v.Minor; \"VersionBuild\" = $v.Build; \"VersionRevision\" = $v.Revision};\n    \"createOnNotFound\" = $CreateOnNotFound;\n    \"merge\" = $Merge; \n    \"publish\" = $Publish     \n} | ConvertTo-Json\n\ntry {\n     Invoke-RestMethod -Method Post -Uri \"https://api.releasenoteshub.com/api/pull/PullVersion/$ProjectId\" -Headers $headers -Body $body -ContentType application/json-patch+json\n} catch {\n    Write-Host \"StatusCode:\" $_.Exception.Response.StatusCode.value__ \n    Write-Host \"StatusDescription:\" $_.Exception.Response.StatusDescription\n    throw $_.Exception\n}"
+  },
+  "Parameters": [
+    {
+      "Id": "3341141f-ab5d-4204-9d84-b46cb0e1ba71",
+      "Name": "ApiKey",
+      "Label": "ApiKey",
+      "HelpText": "https://support.releasenoteshub.com/article/show/105050-how-do-i-get-an-api-key",
```

Comment:
Could you include a one-liner to help what this parameter is, and include the link for when people need further info- markdown is supported.
(This would allow users to only go to another place from Octopus if they still need further info)